src/plugins/home/public/application/components/_home.scss

```diff
@@ -1,16 +1,31 @@
 // Local page vars
-$homePageHeaderHeight: $euiSize * 8;
 $homePageWidth: 1200px;
 
 .homPageHeader {
-  height: $homePageHeaderHeight;
   margin: 0 auto;
   max-width: $homePageWidth;
-  padding: $euiSizeXL $euiSize 0;
+  padding: $euiSizeXL $euiSize $euiSizeXXL;
```

Comment:
I'm wondering if it would be better to set the padding to `$euiSizeXL $euiSize` by default, and then if the below solutions container is rendered/visible, then a modifier class is added to apply additional bottom padding (to account for the overlap of those solution cards).

Also, I imagine we probably want to have some top padding always applied to the `.homPage` element, to ensure that page contents are properly spaced from the header when the solution cards aren't present.

Reply:
The logic for checking solutions exists a layer deeper than the home page, so that presents a challenge. As a simple alternative, I've changed the solutions component (that contains said logic) to output a larger spacer when no solutions exist. This results in the following layout:

<img width="1296" alt="Screenshot 2020-07-27 10 18 22" src="https://user-images.githubusercontent.com/446285/88559683-bee5fc00-cff2-11ea-8dbb-3cb42a94fcb3.png">

The extra padding in the header remains, but it feels less noticeable (to me) with the 'add data' panel pushing down from the header a bit. I think this gets us by for now given this feels like an edge case scenario.